qiskit/transpiler/passes/routing/algorithms/token_swapper.py

```diff
@@ -48,27 +49,23 @@ class ApproximateTokenSwapper(Generic[_V]):
     Internally caches the graph and associated datastructures for re-use.
     """
 
-    def __init__(self, graph: nx.Graph, seed: Union[int, np.random.Generator] = None) -> None:
+    def __init__(self, graph: rx.PyGraph, seed: Union[int, np.random.Generator] = None) -> None:
```

Comment:
```suggestion
    def __init__(self, graph: rx.PyGraph, seed: Union[int, np.random.Generator, None] = None) -> None:
```

Reply:
Also the type hints in the docstring can be removed 🙂

Reply:
Sigh, I will never get used to type hints...